### PR TITLE
Switch to cos7 container

### PR DESCRIPTION
**Pull request by ZimmerA**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This switches the linux 64 container to the newer cos7 container to have acces to glibc >= 2.17. It also enables the building of the recipe.

### CODE REVIEW

**Comment by ZimmerA** [comment]
@conda-forge-admin, please rerender

**Comment by dhirschfeld** [comment]
Ha, beat me to it!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,8 +19,7 @@ source:
   folder: dotnet
 
 build:
-  skip: true  # [osx or win or linux]
-  number: 0
+  number: 1
```
Comment (dhirschfeld) [inline recipe/meta.yaml:22]:
```suggestion
  skip: true  # [osx]
  number: 1
```
I wasn't able to get OSX to work so it should be skipped (for now)

Comment (ZimmerA) [reply recipe/meta.yaml:22]: Yeah! I already realized, thanks for fixing it :D

**Review by dhirschfeld — APPROVED** [review]

**Comment by ZimmerA** [comment]
@conda-forge-admin, please rerender

**Comment by ZimmerA** [comment]
Looks mergeable to me after this build is done. I didn't know if I had to re-render again so I just did.

**Pull request merged**